src/main/dialog/themedwindow.ts

```diff
@@ -114,7 +121,7 @@ export class ThemedWindow {
           <div class="page-container">
             <jlab-dialog-titlebar id="title-bar" data-title="${
               this._window.title
-            }" class="${
+            }" data-closable="${this._closable.toString()}" class="${
```

Comment:
Being explicit with embedding in HTML makes it one less thing to worry about.

```suggestion
            }" data-closable="${this._closable ? 'true' : 'false'}" class="${
```